erica/application/grundsteuer/grundsteuer_input_eigentuemer.py

```diff
@@ -75,4 +76,10 @@ def must_be_set_only_if_two_persons(cls, v, values):
             raise ValueError('has to be set if two persons')
         if values.get('person') and len(values.get('person')) != 2 and v is not None:
             raise ValueError('must not be set if other than two persons')
+        return v
+
+    @validator("person")
+    def must_have_all_test_ids_or_no_test_ids(cls, v, values):
```

Comment:
We're not doing this for Rente yet, right? Should we maybe at least point them to the problem?

Reply:
Yes, let's talk to them